### ISSUE TITLE
Guts within guts: the microbiome of the intestinal helminth parasite Ascaris suum is derived but distinct from its host

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#113]
Guts within guts: the microbiome of the intestinal helminth parasite Ascaris suum is derived but distinct from its host – Ankur Midha et al. – Microbiome 2022

https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01399-5

**Comment by git-terbug** [comment]
Israel Anguiano

**Comment by bolanlew** [comment]
Name: Bolanle Wahab
Hi @SvetlanaUP @cmirzayi i’d like this article assigned to me. Thanks

**Comment by SvetlanaUP** [comment]
Hi @bolanlew, you are already assigned to https://github.com/waldronlab/BugSigDBcuration/issues/121

**Comment by CaramelDesign** [comment]
Hi, @SvetlanaUP, I'll like this article assigned to me. Thanks.

Name: Zaynab A. Ogunnubi.

**Comment by Peacesandy** [comment]
Assigned @CaramelDesign

**Comment by AleruDivine** [comment]
Hello @CaramelDesign, I hope you're having a great day.
I see that this issue has been assigned to you for over 2 weeks, and with this being the final week for contributions, I wanted to check in and see if everything is going well. If you're facing any blockers, or any questions, please do let me know. I would love to help.

Thank you so much!

**Comment by CaramelDesign** [comment]
> Hello @CaramelDesign, I hope you're having a great day. I see that this issue has been assigned to you for over 2 weeks, and with this being the final week for contributions, I wanted to check in and see if everything is going well. If you're facing any blockers, or any questions, please do let me know. I would love to help.
> 
> Thank you so much!

Hello @AleruDivine, Thank you so much for reaching out. 
Yeah, I was the one who got in touch on Slack channel last week when I hit a blocker and you explained how each sites would have a separate experiment. That's what I'm working on.

**Comment by AleruDivine** [comment]
Thats great to hear @CaramelDesign 🤗
Well done! 

I would love to review when you are done, if that is okay with you.

**Comment by CaramelDesign** [comment]
> Thats great to hear @CaramelDesign 🤗 Well done!
> 
> I would love to review when you are done, if that is okay with you.

Yes, that's fine please. Thank you!

**Comment by AleruDivine** [comment]
Hi @CaramelDesign, I hope you are doing alright. I am just checking in to know how your curation is going. If you still have any blockers or if you need a review, just hit me up 🤗

**Comment by Analyst-Joan** [comment]
Hi @CaramelDesign Please can you confirm if this study is being curated at [Study_947](https://bugsigdb.org/Study_947 ) which listed under "studies without experiment" or another link?

**Comment by BarakatAA** [comment]
Hi @CaramelDesign hope you're having a great day. I noticed this study does not yet have a 'needs review' tag.

If you need help or a second opinion on your curation, do reach out. All the best

**Comment by SvetlanaUP** [comment]
@BarakatAA please finalize this curation.

**Comment by BarakatAA** [comment]
Good morning @SvetlanaUP Noted, will do

**Comment by BarakatAA** [comment]
Good morning @SvetlanaUP This study is now ready for review, thank you
https://bugsigdb.org/Study_949

**Comment by Scholarpat** [comment]
Hi @SvetlanaUP,

After reviewing this curation, here are my observations:

For experiments comparing infected vs. non-infected pigs based on the site of infection (experiments 1-5), there was no significant alpha diversity observed, as reported in Fig. 1B. However, there was significant alpha diversity observed for the experiments comparing pigs (infected and uninfected jejunum) to Ascaris microbiomes, as reported in Fig. 1C. Differentially abundant bacteria were only reported for pigs with infected jejunum vs. Ascaris (Fig. 5A and Table S8). This has been curated as experiment 9.

There is another experiment comparing Ascaris worms based on sex (female vs. male). Although the alpha diversity was not significant, differentially abundant bacteria were reported (in Fig. 4D and Table S7) for this experiment. This has been curated as experiment 8.

In summary, experiments 1-7 should not have been curated.

Below are the changes I've made:
- Changed the group definitions for both experiments
- Changed 'Clostridium sp.' initially curated to 'Clostridium' (txID: 1485), which is another name for 'Clostridium sensu stricto' that was originally reported in the article.

Please kindly delete experiments 1 to 7.

The curation is complete. Thank you.

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_949 reviewed.
